src/cmd/services/m3query/config/config.go

```diff
@@ -415,8 +427,14 @@ type RPCConfiguration struct {
 	// ListenAddress is the RPC server listen address.
 	ListenAddress string `yaml:"listenAddress"`
 
-	// RemoteListenAddresses is the remote listen addresses to call for remote
-	// coordinator calls.
+	// RemoteZones are the configuration settings for remote coordinator zones.
+	RemoteZones RemoteZones `yaml:"remoteZones"`
```

Comment:
nit: Can we call this just `RemoteClusters`?  Technically speaking they could also be regions, etc.

Reply:
Second though, maybe just call it `Remotes` and have a `type Remote struct` instead of `type RemoteZone struct`

Reply:
I gotta be honest and say I couldn't decide on a name for this so I went with `RemoteZones` because I knew it would be contentious in the PR haha; `remotes` sounds good